tests/Backend/Mixed/SharingTest.php

```diff
@@ -290,6 +284,46 @@ public function testOrganizationPublicSharing()
         $client->dropBucket($linkedBucketId);
     }
 
+    public function testOnlyTokensHavingCanManagePermissionsCanUnlinkBucket()
```

Comment:
Nedalo se to spojit s testem na org-project sharing? Ten token by v něm vytvořit určitě taky šel. A kdyby to bylo na konci, tak nevadí, že ho dropneš. A nemusel bys znova initovat buckety a tak...

Reply:
upravil jsem to a presunul
ten test pro org-project sharing mi prisel tak dlouhej ze jsem tam nechtel uz nic pridavat. ale jo takhle v PR to vypada hezky jako takhle v PR to vypada hezky

Reply:
Dlouhý testy jsou sice problém, ale na druhou stranu nám šetří čas buildu. A je to 30s tady 30s tamhle a najednou je to 10 minut :/